app/styles/modules/_home.scss

```diff
@@ -0,0 +1,202 @@
+#landing-page {
+  opacity: 0;
+}
+
+/* The following is a lightly modified version of the styles
+*  used by the fullpage plugin.
+*/
+
+#superContainer {
+  height: 100%;
+  position: relative;
+
+/* Touch detection for Windows 8 */
+-ms-touch-action: none;
```

Comment:
Not me, but I did just go and check and remove the vendor prefix here and run `grunt css` and the generated CSS is (wait for it...):
```css
#superContainer {
  height: 100%;
  position: relative;
  -ms-touch-action: none;
      touch-action: none; }
```

Autoprefixer to the rescue! (at least in this situation, I didn't verify with all the `box-sizing` and `transition` styles below)